api/v1alpha1/releaselink_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ReleaseLinkSpec defines the desired state of ReleaseLink
+type ReleaseLinkSpec struct {
+	// DisplayName refers to the name of the Release Link to link a user and managed workspace together.
+	DisplayName string `json:"displayName"`
+
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
```

Comment:
I actually don’t like it but code generated by operator sdk doesn’t add an space between `//` and the tag itself. We should be consistent (even if that means change the generated ones).

Reply:
so you're saying s|// +kube|//+kube| ?